<ui version="4.0" >
 <class>AuxiliaryDataForm</class>
 <widget class="QWidget" name="AuxiliaryDataForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QFrame" name="_outerFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="_fileLabel" >
          <property name="text" >
           <string>Filename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_filename" />
        </item>
        <item>
         <widget class="QPushButton" name="browseButton" >
          <property name="text" >
           <string>Browse ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Label:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_labelEdit" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Rotate:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Angle in degrees:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_rotateAngle" >
          <property name="minimum" >
           <double>-360.000000000000000</double>
          </property>
          <property name="maximum" >
           <double>360.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Axes:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_rotateX" >
          <property name="minimum" >
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum" >
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_rotateY" >
          <property name="minimum" >
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum" >
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_rotateZ" >
          <property name="minimum" >
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum" >
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="_coordinateSystemLabel" >
          <property name="text" >
           <string>Coordinate system for file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_coordinateSystemName" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="coordinateSystem" >
          <property name="text" >
           <string>Choose...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Translate:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_translateX" />
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_translateY" />
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_translateZ" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="_overrideColor" >
          <property name="text" >
           <string>Override color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_colorLabel" >
          <property name="text" >
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QtTools::ColorButton" name="_colorButton" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_transparencyLabel" >
          <property name="text" >
           <string>Transparency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_transparencySpinBox" >
          <property name="maximum" >
           <double>255.000000000000000</double>
          </property>
          <property name="value" >
           <double>255.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Texture:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_textureFilename" />
        </item>
        <item>
         <widget class="QPushButton" name="textureBrowse" >
          <property name="text" >
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QCheckBox" name="_usePointSize" >
          <property name="text" >
           <string>Use point size:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QSpinBox" name="_pointSize" >
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="_useLineWidth" >
          <property name="text" >
           <string>Use line width:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QSpinBox" name="_lineWidth" >
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTools::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global" >QtTools/ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
